js/js-functions/js-functions_functions-with-parameters-extra/js/index.js

```diff
@@ -31,6 +35,9 @@ printSquare(5);
 */
 
 // Uncomment the following function calls and implement the function printRectangleArea
+function printRectangleArea(w, l) {
+  console.log("the area of the rectangle is" + " " + w * l);
```

Comment:
Here you could put the space after the word "is", just for having a cleaner solution:
console.log("the area of the rectangle is " + w * l);

Also could be nice to define first a variable with the calculation and then used it for the console log, just for readability reasons.